src/lib/utils/constants.ts

```diff
@@ -15,6 +17,13 @@ export const enum BrandColors {
 
 export const AmanekoEmojis = {};
 
-export const VTuberOrgs = {};
-
-export const HolodexMembersOnlyPatterns = ['membersonly', 'members only', "member's only", 'member', 'メン限', 'メンバー限定'];
+export const VTuberOrgEmojis = new Map<string, string>([
+	['Hololive', 'Hololive'], //
+	['PhaseConnect', 'PhaseConnect'],
+	['PrismProject', 'PrismProject'],
+	['Nijisanji', 'Nijisanji'],
+	['NijisanjiEN', 'NijisanjiEN'],
```

Comment:
NijisanjiEN should maybe has Nijisanji as value considering it's the same company